test/test_helper.rb

```diff
@@ -3,10 +3,11 @@
 require 'rails/test_help'
 require 'minitest/pride'
 require 'mocha/mini_test'
+require 'capybara/rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.